radar/2025-11-21/cht-stock-monitoring-workflow.md

```diff
@@ -0,0 +1,8 @@
+---
+title:      "cht-stock-monitoring-workflow"
+ring:       trial
```

Comment:
Honestly, this is being used in multiple production instances, so it seems like it could be `adopt.`


```suggestion
ring:       adopt
```